app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class MessagesController < ApplicationController
+      def index
+        @messages = Message.all
+        render json: @messages, status: :ok
+      end
+    end
```

Comment:
- [ ] On the screen I am getting an array of `greetings` objects instead of random `greeting` Please fix that
<img width="813" alt="Screen Shot 2023-06-06 at 12 21 13" src="https://github.com/Lancelot-SO/hello-rails-react/assets/44745855/20b69b1e-71e4-44c8-be8a-6e033ce97635">